base_layer/core/src/proto/transaction.proto

```diff
@@ -75,10 +73,12 @@ message OutputFeatures {
     uint64 maturity = 2;
 
     bytes metadata = 3;
-
+    // Numbering 4-15 taken
```

Comment:
My intention here was to use the recommendation here https://developers.google.com/protocol-buffers/docs/proto3#assigning_field_numbers but I think it will be confusing, so lets just use numbers from 4 onwards

```suggestion
    
```